src/hearings/containers/hearing-actuals/hearing-actuals-timing/hearing-actuals-timing.component.ts

```diff
@@ -17,12 +17,19 @@ import { ValidatorsUtils } from '../../../utils/validators.utils';
 })
 export class HearingActualsTimingComponent implements OnInit, OnDestroy {
 
-  public static timeMatcher = /\d{2}:\d{2}/;
+  public static TIME_MATCHER = /\d{2}:\d{2}/;
+  private static readonly ERROR_MESSAGE_VALID_TIME = 'Enter a valid time.';
```

Comment:
Shall we move the error messages constants to an Enum? You can refer to `HearingDatePriorityEnum` or `HearingJudgeSelectionEnum` in `hearings.enum.ts` file.